client/app/ui-components/tile/tile.component.js

```diff
@@ -135,34 +135,47 @@ angular.module("managerApp")
         scope: true,
         template: `
             <div>
-                <cui-dropdown-menu class="cui-tile__item-button" data-ng-if="$ctrl.hasAvailableAction()">
-                    <cui-dropdown-menu-button>
-                        <ng-include src="'app/ui-components/icons/button-action.html'"></ng-include>
-                    </cui-dropdown-menu-button>
-                    <cui-dropdown-menu-body>
-                        <div class="oui-action-menu">
-                            <div class="oui-action-menu__item oui-action-menu-item" data-ng-repeat="action in $ctrl.actions track by $index">
-                                <div class="oui-action-menu-item__icon"></div>
-                                <a class="oui-button oui-button_link oui-action-menu-item__label" data-ng-if="action.href"
+                <oui-dropdown class="cui-tile__item-button"
+                    data-align="end"
+                    arrow
+                    data-ng-if="$ctrl.hasAvailableAction()">
+                    <button
+                        oui-dropdown-trigger
+                        class="oui-action-menu-button">
+                        <svg xmlns="http://www.w3.org/2000/svg" viewbox="-1 -1 74 74">
```

Comment:
do we really need to use a svh here?

Reply:
I saw an ui-sref a few lines below, so I can't use action-menu component without the  controller...
I chose to use oui-dropdown directly to handle ui-sref. I think it's more error-proof for now but yes, the drawback is that I have to insert an svg in the HTML :/